src/pages/unexpected-error/home/home.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+
+import globalStyles from 'assets/stylesheets/global-styles.module.scss';
+import { AppLink } from 'routes/app-link';
+import { RouteName } from 'routes/routes';
+import { Button } from 'common/button';
+import { Avatar } from 'common/avatar';
+import { Modal } from 'common/modal';
+import { ModalSizes } from 'common/modal/modal';
+import { TextField, TextFieldStatus } from 'common/text-field';
+import { TextArea } from 'common/text-area';
+import { mailFormat } from 'helpers/utils';
+import styles from './home.module.scss';
+import { ReactComponent as MailSVG } from '../../assets/icons/mail.svg';
+import { ReactComponent as CloseSVG } from '../../assets/icons/close.svg';
+
+const Home = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const closeModal: () => void = () => {
+    setOpenModal(false);
+  };
+  const [inputState, setInputState] = useState('');
+  const [textAreaState, settextAreaState] = useState('random text');
+  const [helperIState, setHelperIState] = useState('');
+  const [helperTAState, setHelperTAState] = useState('');
```

Comment:
why this name? jajajaj

Reply:
:stuck_out_tongue_winking_eye::stuck_out_tongue_closed_eyes:

Reply:
@agustinasalmanton This is from the template, I didn't created it